src/Atoms/button.css

```diff
@@ -0,0 +1,18 @@
+.button {
```

Comment:
Try to avoid any extra CSS files for components.
Try to have a component scoped css. (CSS-In-JS)